bugshaker/src/main/java/com/github/stkent/bugshaker/ActivityReferenceManager.java

```diff
@@ -16,24 +16,24 @@
  */
 package com.github.stkent.bugshaker;
 
+import java.lang.ref.WeakReference;
+
 import android.app.Activity;
 import android.os.Build;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
-import java.lang.ref.WeakReference;
-
 public final class ActivityReferenceManager {
 
     @Nullable
-    private WeakReference<Activity> wActivity;
+    private static WeakReference<Activity> wActivity;
 
-    public void setActivity(@NonNull final Activity activity) {
-        this.wActivity = new WeakReference<>(activity);
+    public static void setActivity(@NonNull final Activity activity) {
+        wActivity = new WeakReference<>(activity);
     }
```

Comment:
We should have static methods accessing only variables:
1) created within that method
2) passed in as parameters